### ISSUE TITLE
End-to-End Latency Report Runner

### ISSUE THREAD

**Issue by theodorelu27** [EdgeLab-FHDO/Edge-Diagnostic-Platform#97]
Implement a new component to the Client Side of the application, in which the main role was to submit the end-to-end latency of each iterations to the Master through REST

Components to be added to the Client:

- A new thread runner that would wait for input from the main operator
- A new operator that would handle the send, with REST, the following information: End-to-End Latency, Server Usage, Server ID, timestamp

Components to be added to the Master:

- REST input for the information
- Module to receive the input, process, and output into format that can be processed in MathLab

**Comment by alyhasansakr** [comment]
@theodorelu27 I agree on the client part, but I believe the master part should be already possible with what we currently have. That is to receive a REST input (as JSON) and write the value to a file with the file writer module. This file can then be processed in MatLab.

**Comment by theodorelu27** [comment]
@alyhasansakr ah okay, thanks for confirming
I'd update the requirement to "REST Input" and "Filewriter output", these two would be implemented as a connections pair in the configuration file
